ONSdigital/census-rm-ddl
# RMC-173 Add blank questionnaire column to case uac qid link table

# Motivation and Context
For the blank questionnaire feature we need a flag on the UAC QID link to tell us if it was returned blank.

# What has changed
* Update groundzero and patch to add new column 
* Update patch number and version tag in ground zero
* Update gitignore with a java template

# Checklist
Reminder: Make sure to version tag this after the PR is merged
* [x] Updated patch number
* [x] Updated version_tag

# How to test?
Bring the database up locally on master docker dev then run the latest patch script. Then checkout the service branches linked, switch them to hibernate verify in docker dev config and check they start up. Then delete the schemas and recreate them from the groundzero, again the branch services should start up in verify mode.

# Links
https://trello.com/c/EmrziSj3/439-rmc-173-blank-questionnaire-returned-21
https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/38
https://github.com/ONSdigital/census-rm-case-processor/pull/128
https://github.com/ONSdigital/census-rm-ddl/pull/47
https://github.com/ONSdigital/census-rm-pubsub/pull/40
https://github.com/ONSdigital/census-rm-acceptance-tests/pull/222
